utils/formatDate.ts

```diff
@@ -91,6 +91,13 @@ export function isValidDate(dateString: ApiGatewayDateString): boolean {
   }
 }
 
+/**
+ * Returns true if the dateString is Jan 1, year 1.
+ */
+export function isNullDateString(dateString: ApiGatewayDateString): boolean {
+  return dateString === '0001-01-01T00:00:00'
+}
```

Comment:
maybe a nitpick, but I thought this would check `'0001-01-01T00:00:00' | null` based on the name - should it be that? That would also let you simplify the call in `getScenarioContent`

Reply:
Not necessary, but what if this was isPre2020DateString instead? it seems like you've added all the other code for that already in this PR. and the TODO below could be just for the logging part.

Reply:
I think this should actually be turned into something like `isDateStringFalsy()` because the logic for Scenario 1 is that `scheduleDate` must be null. To be extra safe, I'm checking for falsy (empty string, null, undefined) and want to extend `0001-01-01T00:00:00` to be included in the definition of "falsy" in this case.